.travis.yml

```diff
@@ -47,20 +45,24 @@ matrix:
     # Python tests
     - os: linux
       env: SUITE=pytest
-    - os: osx
-      env: SUITE=pytest
-      install: easy_install virtualenv
 
     # Integration tests
     - os: linux
-      env: SUITE=integration-test
-    - os: osx
-      env: SUITE=integration-test
-      install: easy_install virtualenv
+      env: PYTHON=python3.7 SUITE=integration-test
```

Comment:
_nit_: To simplify the setup, can we have the Makefile ensure a venv with the correct python version and instead use `language: rust` here? This should also make it easier to reproduce tests locally.